adapters/access.go

```diff
@@ -497,7 +496,7 @@ func (a *AccessAdapter) GetNodeVersionInfo(
 	*access.NodeVersionInfo,
 	error,
 ) {
-	return nil, fmt.Errorf("not supported")
+	return &access.NodeVersionInfo{}, nil
```

Comment:
Maybe we could include the `flow-go` semver/commit?  May be useful to have in the future.